src/components/EntryListing/EntryListing.js

```diff
@@ -57,7 +57,7 @@ export default class EntryListing extends React.Component {
           : inferedFields.remainingFields && inferedFields.remainingFields.map(f => (
             <p key={f.get('name')} className={styles.cardList}>
               <span className={styles.cardListLabel}>{f.get('label')}:</span>{' '}
-              { entry.getIn(['data', f.get('name')], '').toString() }
+              { entry.getIn(['data', f.get('name')], '')?entry.getIn(['data', f.get('name')], '').toString():'' }
```

Comment:
We can improve readability plus shorten a bit: `(entry.getIn(['data', f.get('name')]) || '').toString()`.